TACC/Core-Portal
# Bug/fp 1428 job history error

## Overview: ##

Job history item crashes client

## Related Jira tickets: ##

* [FP-1428](https://jira.tacc.utexas.edu/browse/FP-1428)

## Summary of Changes: ##
- Direct link to a job history item throws an error and crashes client
- StyleName element is not defined

## Testing Steps: ##
1. Go to https://cep.dev/workbench/history/jobs and clic on a job 'View Details' link
2. Copy the link of that particular job and paste it on a different browser tab
3. The modal window with the job information should be shown

## UI Photos:

<img width="1389" alt="Screen Shot 2022-02-09 at 3 22 36 PM" src="https://user-images.githubusercontent.com/62672123/153292615-38a81d23-3d64-4de0-86b5-e86a76f5c361.png">

<img width="1084" alt="Screen Shot 2022-02-09 at 3 23 38 PM" src="https://user-images.githubusercontent.com/62672123/153292641-0a72cd94-bf46-48b1-a0c2-a75e4e353b99.png">



## Notes: ##
